### PR TITLE
core: Add support for all available output parsers in _load_output_parser()

### PR DESCRIPTION
**Pull request by DougTrajano**
In this PR, we are updating the `_load_output_parser()` function to dynamically get all available output parsers to be loaded.

I am not 100% comfortable with this solution, because I am accessing a protected property outside the class (_type) in line 81, so feel free to suggest changes. :)

### CODE REVIEW

**Comment by DougTrajano** [comment]
@baskaryan

**Comment by DougTrajano** [comment]
@baskaryan, @eyurtsev, @hwchase17

**Comment by DougTrajano** [comment]
@efriis @eyurtsev can you review this PR?

File: libs/core/langchain_core/prompts/loading.py [hunk 1]
```diff
@@ -72,13 +73,24 @@ def _load_examples(config: dict) -> dict:
...
+    return parsers_map
+
+
 def _load_output_parser(config: dict) -> dict:
     """Load output parser."""
     if "output_parser" in config and config["output_parser"]:
+        output_parsers_map = _get_output_parsers_map()
         _config = config.pop("output_parser")
         output_parser_type = _config.pop("_type")
-        if output_parser_type == "default":
```
Comment (baskaryan) [inline libs/core/langchain_core/prompts/loading.py:80]: we probably still want to support output_parser_type == "default"

Comment (DougTrajano) [reply libs/core/langchain_core/prompts/loading.py:80]: Ok, I will add it.

Comment (DougTrajano) [reply libs/core/langchain_core/prompts/loading.py:80]: Done @baskaryan

**Comment by DougTrajano** [comment]
@eyurtsev @baskaryan, @eyurtsev, @hwchase17 ping

**Comment by hwchase17** [comment]
this is using the old way of serializing object. there is a new serialization schema, if we want to support output parsers we should do it there, but it may be some work

**Pull request closed without merging**